gp-includes/things/translation.php

```diff
@@ -81,6 +81,11 @@ public function fix_translation( $translation ) {
 
 	public function restrict_fields( $translation ) {
 		$translation->translation_0_should_not_be( 'empty_string' );
+		$translation->translation_1_should_not_be( 'empty_string' );
+		$translation->translation_2_should_not_be( 'empty_string' );
+		$translation->translation_3_should_not_be( 'empty_string' );
+		$translation->translation_4_should_not_be( 'empty_string' );
+		$translation->translation_5_should_not_be( 'empty_string' );
```

Comment:
Perfect, does it work on the display code below?  That code seems to be comparing against an empty string and not NULL...